.github/workflows/stylecheck-prs.yml

```diff
@@ -0,0 +1,25 @@
+name: stylecheck
+on:
+  pull_request:
+    branches: [master]
+
+jobs:
+  stylecheck:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+      - name: get Changed Files
+        id: get_changed_files
+        uses: lots0logs/gh-action-get-changed-files@2.1.4
+        with:
+          token: ${{ github.token }}
```

Comment:
Based on the docs, GH uses `${{ secrets.GITHUB_TOKEN }}` everywhere they need it. Would you mind changing this and the other usage of it to `${{ secrets.GITHUB_TOKEN }}` ?

Reply:
sure!